src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/DispatchBusinessProcessCallbackHandler.java

```diff
@@ -36,17 +40,19 @@
     }
 
     private CallbackResponse dispatchBusinessProcess(CallbackParams callbackParams) {
-        Map<String, Object> data = callbackParams.getRequest().getCaseDetails().getData();
-        BusinessProcess businessProcess = callbackParams.getCaseData().getBusinessProcess();
+        CaseData caseData = callbackParams.getCaseData();
+        BusinessProcess businessProcess = caseData.getBusinessProcess();
+        CaseData.CaseDataBuilder caseDataBuilder = caseData.toBuilder();
         if (businessProcess.getStatus() == READY) {
-            data.put("businessProcess", BusinessProcess.builder()
-                .camundaEvent(businessProcess.getCamundaEvent())
-                .status(DISPATCHED)
-                .build());
+            caseDataBuilder
+                .businessProcess(BusinessProcess.builder()
+                                     .camundaEvent(businessProcess.getCamundaEvent())
+                                     .status(DISPATCHED)
+                                     .build());
```

Comment:
good example of why businessProcessService is now not needed

Reply:
also another use case for some void methods to update business process in case data directly rather than need case data builder